src/kfactory/pcells/dbu/waveguide.py

```diff
@@ -1,6 +1,7 @@
 from typing import Optional
 
 from ... import KCell, LayerEnum, autocell, kdb, klib
+from ...pdk import _ACTIVE_PDK
```

Comment:
Why? The waveguide doesn't care about the pdk. It can use it, but it does not depend on it

Reply:
Edited